src/test/java/uk/gov/ons/ctp/response/action/service/PrintFileServiceTest.java

```diff
@@ -38,8 +38,7 @@
 
   @Test
   public void testConvertToPrintFile() {
-    List<ActionRequestInstruction> actionRequestInstructions =
-        ObjectBuilder.buildListOfActionRequests();
+    List<ActionRequest> actionRequestInstructions = ObjectBuilder.buildListOfActionRequests();
```

Comment:
actionRequestInstructions aren't a thing anymore, so this variable name should be changed.  Same with the other one in this file.

Reply:
done 👍